build/windows/fix_archives.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -e
+#
+#
+# Gets dist/zip_dirty created by Goreleaser and reorganize inside files
+#
+#
+PROJECT_PATH=$1
+
+for zip_dirty in $(find dist -regex ".*_dirty\.\(zip\)");do
```

Comment:
The `for...find` construct is problematic and might have unforeseen effects for certain files, specially those with spaces.

For example:

```
[~/Desktop]$ ls -l test
total 0
-rw-r--r--  1 rsantalla  staff  0 Jan 12 12:36 a b
-rw-r--r--  1 rsantalla  staff  0 Jan 12 12:37 c
-rw-r--r--  1 rsantalla  staff  0 Jan 12 12:37 d
[~/Desktop]$ for t in $(find test -type f); do echo x $t x; done
x test/c x
x test/d x
x test/a x
x b x
```
`find | while read ...` is almost a drop-in replacement and shouldn't have issues with IFS:
```
[~/Desktop]$ find test -type f | while read t; do echo x $t x; done
x test/c x
x test/d x
x test/a b x
```